bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminClientWrapper.java

```diff
@@ -196,23 +197,34 @@ public Table apply(com.google.bigtable.admin.v2.Table tableProto) {
   /** {@inheritDoc} */
   @Override
   public void dropRowRange(String tableId, String rowKeyPrefix) {
-    DropRowRangeRequest requestProto = DropRowRangeRequest.newBuilder()
+    DropRowRangeRequest.Builder dropRequestProtoBuilder = DropRowRangeRequest.newBuilder()
         .setName(instanceName.toTableNameStr(tableId))
-        .setRowKeyPrefix(ByteString.copyFromUtf8(rowKeyPrefix))
-        .build();
+        .setDeleteAllDataFromTable(true);
```

Comment:
Please remove `setDeleteAllDataFromTable` from here, and move it to the else statement of the if below.

Reply:
Please also create a new method for creating a `DropRowRangeRequest` that's reusable for `dropRowRange` and `dropRowRangeAsync`

Reply:
Added `AdminWrapper#buildDropRowRangeRequest` for the same.